src/TankController.cpp

```diff
@@ -158,25 +161,31 @@ void TankController::handleUI() {
 /**
  * This is one of two public instance functions.
  * It is called repeatedly while the board is on.
- * (It appears to be called about once every 15 ms.)
  */
 void TankController::loop(bool report_loop_delay) {
   static unsigned long lastTime = 0;
   unsigned long thisTime = millis();
   if (report_loop_delay && lastTime && thisTime - lastTime > 500) {
-    // report unusual delay
     serial(F("unexpected delay of %i ms"), thisTime - lastTime);
```

Comment:
I think it would be clearer to measure the difference between `currentLoopStart` and `previousLoopStop`

Reply:
This would also be a good place to measure and report the difference between `currentLoopStart` and `previousLoopStart`